src/state.rs

```diff
@@ -271,6 +276,31 @@ impl StateSet {
     pub fn iter(self) -> impl Iterator<Item = State> {
         self.0.iter()
     }
+
+		/// Create a StateSet, with all states active.
```

Comment:
I am not sure why would anyone need this.

Reply:
Even just as a debug mechanism, showing a number or a bitset would be a lot less clear than a list of the states.